tests/perf-system/submitter/CMakeLists.txt

```diff
@@ -6,10 +6,8 @@ add_executable(
 )
 target_compile_definitions(submit PRIVATE CCF_LOGGER_NO_DEPRECATE)
 
-add_library(stdcxxhttp_parser.host "${HTTP_PARSER_SOURCES}")
-target_compile_definitions(
-  stdcxxhttp_parser.host PRIVATE CCF_LOGGER_NO_DEPRECATE
-)
+add_library(stdcxxhttp_parser "${HTTP_PARSER_SOURCES}")
```

Comment:
We should be able to avoid building those by disabling them when libcxx is enabled. That's now only in instrumented builds, and we don't need the perf tooling there. We can save on regular builds though.